Automattic/wp-calypso
# Revert "Signup Framework : Cleanup dead stepcode..." #80352

Related to p1694760534062509-slack-C02FMH4G8

## Proposed Changes

* This reverts commit 2e69dd96d6c84b2c7fad1e49dc31d11550223287. ( #80352 )

## Testing Instructions


* Visit /start/domain/domain-only?new=testtestsv&search=yes 
* Click "Select" button on the testtestsv.com Entry 
* Click "Get Domain"

Before patch: See 
```
Uncaught Error: This step (themes) provides an unspecified dependency [themeSlugWithRepo]. Make sure to specify it in /signup/config/steps-pure.js, using the providesDependencies property.
```

After patch: Works
